public/controllers/management/components/management/statistics/statistics-overview.js

```diff
@@ -90,6 +92,18 @@ export class WzStatisticsOverview extends Component {
         clusterNodes: [],
         clusterNodeSelected: 'all',
       });
+
+      const options = {
+        context: `${WzStatisticsOverview.name}.componentDidMount`,
+        level: UI_LOGGER_LEVELS.ERROR,
+        severity: UI_ERROR_SEVERITIES.BUSINESS,
+        error: {
+          error: error,
+          message: error.message || error,
+          title: error.name || error,
+        },
+      };
```

Comment:
**issue**: The `error` variable in the `try/catch` block is called as `err`

**suggestion**: Rename the variable in the `catch` parameter to `error` instead of `err`

Reply:
Thanks @Desvelao. Resolved in commit.